src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
+
+const { ExpenseServises } = require('../services/expenses.service');
+const { UserServices } = require('../services/users.service');
+const { STATUS_CODE } = require('../utils/constants');
+
+const getExpenses = (req, res) => {
+  const { userId, from, to, categories } = req.query;
+
+  let expenses = ExpenseServises.getExpenses();
+
+  if (userId) {
+    expenses = expenses.filter(expense => expense.userId === Number(userId));
+  }
+
+  if (categories) {
+    const categoriesIsArray = Array.isArray(categories)
```

Comment:
But not really good variable name
Avoid using words `array` `object`... in variable name